pom.xml

```diff
@@ -17,8 +17,8 @@
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Dashboard+View</url>
 
   <properties>
-    <jenkins.version>1.554.1</jenkins.version>
-    <findbugs.failOnError>false</findbugs.failOnError> <!-- TODO fix -->
+    <jenkins.version>1.580.1</jenkins.version>
```

Comment:
Given this is still a quite old version, I don't think the bumping is a problem.